reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UptimeMillis.kt

```diff
@@ -1,6 +1,6 @@
 package com.badoo.reaktive.utils
 
-import kotlin.browser.window
+import kotlin.js.Date
 
 internal actual val uptimeMillis: Long
-    get() = window.performance.now().toLong()
+    get() = Date.now().toLong()
```

Comment:
What's the problem with `window.performance.now`? Looks like it's available in all modern browsers https://caniuse.com/#search=performance.now
If we use Date then uptimeMillis will not be monotonic.

Reply:
But not in NodeJS as I can see.
https://stackoverflow.com/questions/23003252/window-performance-now-equivalent-in-nodejs

Reply:
We use NodeJS to run our tests.

Reply:
Let's use https://nodejs.org/api/process.html#process_process_hrtime_time for nodejs then and window.performance.now for browsers

Reply:
Fixed pls check